src/site/_includes/components/EventTable.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// const path = require('path');
+const {html} = require('common-tags');
+// const prettyDate = require('../../_filters/pretty-date');
+// const stripLanguage = require('../../_filters/strip-language');
+// const md = require('../../_filters/md');
+// const constants = require('../../_utils/constants');
+// const getSrcsetRange = require('../../_utils/get-srcset-range');
+// const postTags = require('../../_data/postTags');
```

Comment:
should these be deleted?

Reply:
Yup. I did it in the `web-event-time` PR.